common/src/main/java/tachyon/network/protocol/RPCErrorResponse.java

```diff
@@ -44,17 +40,26 @@ public static RPCErrorResponse decode(ByteBuf in) {
   }
 
   @Override
-  public int getEncodedLength() {
-    // 1 short (mStatus)
-    return Shorts.BYTES;
+  public void encode(ByteBuf out) {
+    out.writeShort(mStatus.getId());
   }
 
   @Override
-  public void encode(ByteBuf out) {
-    out.writeShort(mStatus.getId());
+  public int getEncodedLength() {
+    // 1 short (mStatus)
+    return Shorts.BYTES;
   }
 
   public Status getStatus() {
     return mStatus;
   }
+
+  public Type getType() {
+    return Type.RPC_ERROR_RESPONSE;
+  }
+
+  @Override
+  public String toString() {
+    return "RPCErrorResponse(" + getType().toString() + ", " + mStatus + ")";
```

Comment:
Thanks for the update! I think getting rid of the "getType().toString()" part would be less redundant, since "RPCErrorResponse" should already represent the type.